owncloudApp/src/test/java/com/owncloud/android/presentation/viewmodels/settings/SettingsSecurityViewModelTest.kt

```diff
@@ -139,27 +140,27 @@ class SettingsSecurityViewModelTest : ViewModelTest() {
 
     @Test
     fun `set pref is security enforced enabled - ok - true`() {
-        every { contextProvider.getBoolean(any()) } returns true
+        every { contextProvider.getInt(any()) } returns LockEnforcedType.DIALOG_OPTION.ordinal
 
         securityViewModel.isSecurityEnforcedEnabled().apply {
             assertTrue(this)
         }
 
         verify(exactly = 1) {
-            contextProvider.getBoolean(R.bool.lock_enforced)
+            contextProvider.getInt(R.integer.lock_enforced)
         }
```

Comment:
`verify` clause is not needed in these cases, they do not assert anything relevant from my point of view.

Reply:
Okay, removed.